src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -111,7 +112,8 @@ const xScale = {
   domain: [0, CHART_TRADE_SIZE_MAX_USD],
 }
 
-const FeeChart: React.FC<FeeChartProps> = ({ foxHolding, tradeSize, parameters }) => {
+const FeeChart: React.FC<FeeChartProps> = ({ foxHolding, tradeSize, feeModel }) => {
+  const parameters = feeCurveParameters[feeModel]
   const { FEE_CURVE_MAX_FEE_BPS } = parameters
```

Comment:
nitpick, no need for intermediary variable really
```suggestion
  const { FEE_CURVE_MAX_FEE_BPS } = feeCurveParameters[feeModel]
```

Reply:
7df180f6480d8f1122362daf6bed10b1ef94e7e6

Reply:
https://github.com/shapeshift/web/commit/8a7941349ad24fa8de9b72cc0732fa3da4cfb7a8